### ISSUE TITLE
About the loss

### ISSUE THREAD

**Issue by NJUST-KMG** [ZihaoWang-CV/CAMP_iccv19#9]
Hi, thanks for sharing! I met a problem when I tried to train the cross attention model on flickr30k dataset that the BCE loss was always around 177.4. After I changed the loss function to Ranking loss, it always  25.6. I change the batchsize from 128 to 64, bs from 5 to 1 due to conditions. The rest of parameters is same as you published in config_256.yaml. But I am wondering whether the environment causes this problem. And my experiment is done in python 3.6.13 and torch 1.7.1. Could you tell me the true version and give me some advice?

**Comment by ZihaoWang-CV** [comment]
Sorry for my late reply.
It seems that a PyTorch version later than 1.7 may cause that problem. I'm trying to detect and fix the issue.
